ui/app/styles/core/icon.scss

```diff
@@ -57,3 +57,33 @@ $icon-dimensions-large: 2rem;
     color: $grey;
   }
 }
+
+// Flight Icons general override: notes:
```

Comment:
I don't feel great about this but it seems necessary given Nomad's current status in a transitory period between a its historical UI design and a more Helios-centered one.

Reply:
agreed it's not ideal as we would prefer these class names to be more of a private API, however, you're not the first (and won't be the last) to hook into them for specific situations that arise in our products.